tests/samplers_tests/tpe_tests/test_sampler.py

```diff
@@ -643,7 +643,7 @@ def test_sample_independent_handle_unsuccessful_states(state: optuna.trial.Trial
         trial = frozen_trial_factory(i, dist=dist)
         study._storage.create_new_trial(study._study_id, template_trial=trial)
     trial = frozen_trial_factory(30)
-    sampler = TPESampler(n_startup_trials=5, seed=0)
+    sampler = TPESampler(n_startup_trials=5, seed=2)
```

Comment:
Ditto, and also for the following seed changes.

Reply:
This is a similar situation to the `prior_weight` problem described earlier.

For `seed=0`, even if the states of the inserted trials are different, the center of the Gaussian sampling will be the point corresponding to the prior, and the result of the suggestion will be the same. By using different seed values, the center of the Gaussian sampling can be set to the point corresponding to the inserted trials, and the results can be reproduced differently depending on the states.

Reply:
Wasn't the `master` behavior kept in https://github.com/optuna/optuna/pull/2618/commits/a26b3a77bc302e4c9755a3350efabc5f01eed944 ?

Reply:
No, it was not. The reason is that the order of `mus`, `sigmas`, and `weights`. In the master branch, they are sorted with ascending order of `mus`, but in this PR (multivariate TPE), they are ordered as observed. It affects the sampling order in `ParzenEstimator.sample` and results in the need for a fix in test cases.

Reply:
Cc: @keisuke-umezawa